wrapper/src/main/java/software/amazon/jdbc/HikariPooledConnectionProvider.java

```diff
@@ -140,7 +140,7 @@ public HikariPooledConnectionProvider(
   public boolean acceptsUrl(
       @NonNull String protocol, @NonNull HostSpec hostSpec, @NonNull Properties props) {
     final RdsUrlType urlType = rdsUtils.identifyRdsType(hostSpec.getHost());
-    return RdsUrlType.RDS_INSTANCE.equals(urlType);
+    return urlType.isRds();
```

Comment:
What would be a connection pool key in case of cluster endpoint?

Reply:
The key would be similar to an instance endpoint. PoolKey.url would be the cluster URL and PoolKey.extraKey would be the user (unless a custom HikariPoolMapping is set)

Reply:
I can assume that in case of reader cluster endpoint such connection pool will contain connections to different nodes. What should we do with such connections in case of failover?

Reply:
My testing shows that all connections that existed before failover are broken after failover (regardless of any instance URL or cluster URL used) so I think all connection pools will need to be closed.